ci/fih_test_docker/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN \
 # Clone TF-M and dependencies
 RUN mkdir -p /root/work/tfm &&\
     cd /root/work/tfm  &&\
-    git clone https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git -b TF-Mv1.2-RC1 &&\
```

Comment:
@utzig , @RcColes What I am supposed to do with this line now?

Reply:
I assume wait for @RcColes to submit a PR with the new trusted-firmware-m tag!

Reply:
Yes, just working on getting the new TF-M commit hash now. Ideally should be by end of today. Do you think it'd be better to have it as a separate commit/PR or is it fine to just add it in here?

Reply:
> Do you think it'd be better to have it as a separate commit/PR or is it fine to just add it in here?

A separate PR would be better, but both are OK IMO.

Reply:
PR created at #903

Reply:
@RcColes Looks like after merging that Travis on master failed as this PR CI before fix.
Can you help with that?